hangman_game.py

```diff
@@ -1 +1,42 @@
-# Hangman game
\ No newline at end of file
+# Hangman game
+import random
+import os
+import sys
+states = ["andrapradesh", "bihar", "chattisgarh", "karnataka", "kerala",
```

Comment:
Good try.
Please give a hint on what to guess for a user and how many wrong chances they have.
Basically print the game rules